decidim-core/lib/decidim/content_processor.rb

```diff
@@ -57,7 +57,7 @@ def self.parse(content, context)
     end
 
     # Public: Calls the specified processors to process the given content with
-    # it. For example, to convert hashtags to its Global ID representation.
+    # it. For example, to convert images to its Global ID representation.
```

Comment:
Images aren't converted
```suggestion
    # it. For example, to convert user mentions to its Global ID representation.
```